bindings/ruby/ext/ruby_whisper_params.c

```diff
@@ -1271,6 +1300,7 @@ init_ruby_whisper_params(VALUE *mWhisper)
   DEFINE_PARAM(suppress_blank, 8)
   DEFINE_PARAM(suppress_nst, 9)
   DEFINE_PARAM(token_timestamps, 10)
+  DEFINE_PARAM(max_len, 35)
```

Comment:
Can you re-number params for maintainability? The position of `max_len` is okay and keep it here.

Reply:
Ah yep I was wondering what was appropriate here